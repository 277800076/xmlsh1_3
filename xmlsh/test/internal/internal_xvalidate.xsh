# test of xinclude

# Basic test
echo Validating with DTD
xvalidate  -dtd ../../samples/data/books.dtd < ../../samples/data/books.xml || echo Validation failed

echo Validating with xsd
xvalidate  -xsd ../../samples/data/books.xsd  ../../samples/data/books.xml || echo Validation failed

echo Skipping validating with rng - requires msv library
echo validate  -rng ../../samples/data/books.rng  ../../samples/data/books.xml || echo Validation failed

echo Following should fail
xvalidate  -xsd ../../samples/data/books.xsd  ../../samples/data/othello.xml  2>/dev/null || echo Successfully trapped invalid schema
