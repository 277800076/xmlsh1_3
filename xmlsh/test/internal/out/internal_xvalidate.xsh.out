Validating with DTD
Validating with xsd
Skipping validating with rng - requires msv library
validate -rng ../../samples/data/books.rng ../../samples/data/books.xml
Following should fail
Successfully trapped invalid schema
